README.md

```diff
@@ -631,6 +631,14 @@ This function deletes a specified env entirely. It takes env id `eid` as input.
 > **Note**: `delete_env` is deletes all data for an environment and is IRREVERSIBLE. Do not use unless you absolutely want to remove an environment.
 
 
+#### vis.fork
```

Comment:
I think it would be more appropriate to use `fork_env` as we may have something specifically for forking layouts in the future. CamelCase locations should then use `ForkEnv`.